awx/main/models/inventory.py

```diff
@@ -175,6 +175,13 @@ class Meta:
         related_name='inventory_labels',
         help_text=_('Labels associated with this inventory.'),
     )
+    prevent_instance_group_fallback = models.BooleanField(
+        default=False,
+        help_text=(
+            "If enabled, the job template will prevent adding any inventory or organization "
+            "instance groups to the list of preferred instances groups to run on."
```

Comment:
This help text isn't adding up for me b/c it's a field on `Inventory` model here (I think). I'd like some clarification of what this means in the context of setting it on inventory.

Reply:
help text updated.